webpacking or package.json dependencies broken

**Repro**: Here's a script you can try running in project root directory:
```
#!/bin/sh

mkdir /tmp/pack-test
npm pack
mv streamr-client-*.tgz /tmp/pack-test
pushd /tmp/pack-test
npm init -y
npm install streamr-client-*.tgz
node -e "require('./node_modules/streamr-client')"
popd
rm -rf /tmp/pack-test
```

**Expected**: does the install and completes without errors
**Actual**: After installs:
```
internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module 'babel-runtime/core-js/object/assign'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/private/tmp/pack-test/node_modules/streamr-client/dist/streamr-client.js:1:2235)
    at r (/private/tmp/pack-test/node_modules/streamr-client/dist/streamr-client.js:1:420)
    at Object.<anonymous> (/private/tmp/pack-test/node_modules/streamr-client/dist/streamr-client.js:1:12291)
    at r (/private/tmp/pack-test/node_modules/streamr-client/dist/streamr-client.js:1:420)
    at /private/tmp/pack-test/node_modules/streamr-client/dist/streamr-client.js:1:1219
    at /private/tmp/pack-test/node_modules/streamr-client/dist/streamr-client.js:1:1229
```

This is because babel-runtime isn't in dependencies (it's a devDependency though) but still webpack generates it into `dist`.

**Version**: 2.1.0
Also applies to: 2.1.0-beta.0, 2.0.3, 2.0.0, 2.0.0-beta.1
Does not apply to: 1.0.2

**Suggest** making this script run before npm publish, in a hook somehow, to sanity-check that npm package isn't broken.

--- a/package.json
+++ b/package.json
@@ -42,7 +42,6 @@
     "babel-preset-2017": "0.0.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-stage-2": "^6.24.1",
-    "babel-runtime": "^6.26.0",
     "chai": "^4.1.2",
     "eslint": "^4.19.1",
     "eslint-config-airbnb": "^16.1.0",
@@ -61,6 +60,7 @@
     "webpack-cli": "^3.3.2"
   },
   "dependencies": {
+    "babel-runtime": "^6.26.0",
     "debug": "^3.1.0",
     "ethers": "^4.0.27",
     "eventemitter3": "^3.0.1",

